tests/queries/0_stateless/02884_string_distance_function.reference

```diff
@@ -0,0 +1,37 @@
+0
+1
+7
+7
+10
+3
+6
+3
+10
+6
+6
+3
+10
+1
```

Comment:
Someone fixing expected results will not have too much fun with associating 37 consecutive numbers with the original statements. Consider adding separators a la `SELECT 'const arguments';`.